packages/trader/src/Modules/Account/Sections/Security/ConnectedApps/data-table-template.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import moment from 'moment';
+import { Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const getConnectedAppsColumnsTemplate = handleToggleModal => [
+    { title: 'Name', col_index: 'name' },
```

Comment:
localize on titles?

Reply:
@arash-fs localize is still missing though?

Reply:
hum :thinking: maybe the merge conflict issue?
added it again.